tests/features/contact_form.feature

```diff
@@ -29,8 +27,6 @@ Feature: Contact Form
       | contact_form |
     When I am on "contact"
     Then I should see "Contact - European Commission" in the "title" tag
-    When I fill in "Your name" with "Chuck"
-    And I fill in "Your e-mail address" with "chuck.norris@improbabledommainname.com"
```

Comment:
The tests are incomplete because now, if the user is logged, the test must ensure that any authenticated user cannot type a name and an e-mail in the form and that he views its data instead.